.gitignore

```diff
@@ -6,3 +6,4 @@
 /coverage
 .DS_Store
 .vscode/
+plugins/*/*/options.json
```

Comment:
Can't we just use env variables?

Reply:
No need for this anymore, I'll remove instead